profiles/seccomp/seccomp.go

```diff
@@ -11,7 +11,11 @@ import (
 
 // Seccomp represents the config for a seccomp profile for syscall restriction.
 type Seccomp struct {
-	DefaultAction specs.LinuxSeccompAction `json:"defaultAction"`
+	DefaultAction    specs.LinuxSeccompAction `json:"defaultAction"`
+	DefaultErrnoRet  *uint                    `json:"defaultErrnoRet,omitempty"`
```

Comment:
Why the pointer? Is 0 taken?

Reply:
LOL, I had the same question at the time; https://github.com/opencontainers/runtime-spec/pull/1042#discussion_r423791078

Reply:
#42642